index.js

```diff
@@ -19,5 +19,24 @@ module.exports = function(source) {
         this.cacheable();
     }
 
+    // if force sourcemap requested embed inline sources (helps resolve issues with
+    // transpiled languages like typescript
+    if (userOptions['force-sourcemap']) {
```

Comment:
When you say, "standalone source maps alongside," do you mean source maps that aren't inlined but are in, for instance, a `.js.map` file?